@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageTSDHandleControl.sqf

```diff
@@ -50,4 +49,20 @@ switch (_state get "subPageVarPage" select 0) do {
     case 4: { // THRT
         _this call fza_mpd_fnc_tsdThrtHandleControl;
     };
+};
+
+//Tsd Scale Control
+switch (_control) do {
+    case "r1": {
+        _this call fza_mpd_fnc_tsdScaleHandleControl;
+    };
+    case "r2": {
+        _this call fza_mpd_fnc_tsdScaleHandleControl;
+    };
+    case "r3": {
+        _this call fza_mpd_fnc_tsdScaleHandleControl;
+    };
+    case "r4": {
+        _this call fza_mpd_fnc_tsdScaleHandleControl;
```

Comment:
Good separation of concerns on this, putting the scale into a separate function

Reply:
Maybe instead of having this at the end, put it before the rest of the options and make `tsdScaleHandleControl` return a boolean if it did something with a control or not. Then if it did, you `exitWith` because you know it's been handled`